flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml

```diff
@@ -1099,7 +1099,6 @@
             </UseTemplate>
         </Component>
   </Template>
-
     <Template Name="A32NX_Push_BARO_Template">
```

Comment:
**question**: Is this template still used elsewhere?

Reply:
Yes, it's used for the other push buttons on the standby indicator (no press and hold functionality for these for now)